psi4/src/psi4/libfock/jk.h

```diff
@@ -706,6 +706,34 @@ class PSI_API DirectJK : public JK {
     /// ERI Sieve
     std::shared_ptr<ERISieve> sieve_;
 
+    /// Options object
+    Options& options_;
+
+    // Perform Density matrix-based integral screening?
+    bool density_screening_;
+
+    // => Incremental Fock build variables <= //
+    /// Perform Incremental Fock Build for J and K Matrices?
+    bool incfock_;
```

Comment:
Would it be better to change the name of this variable to "do_incfock_"? It seems that the do_ at the beginning of the variable name is something of a convention for variables like this (e.g., do_csam_, do_J_). Also, it would probably be wise to put in the comment what this variable defaults to.

Reply:
Good idea, but at this point, changing the associated variable would be too much of a "shotgun surgery"